packages/react-ui-ag/src/Carousels/__tests__/ListingCarousel-test.js

```diff
@@ -176,6 +176,13 @@ const listingProps = {
     dimensions: '280-120',
     disableSwipe: true,
   },
+  /* eslint-disable max-len */
+  // TODO
+  // gaa: This needs to be false because jsdom is providing a mocked DOM and react-lazyload
+  // manipulates the DOM so it crashes when attempting to create placeholders
+  // I'll need to look into this further later to see if we can fix this.
+  lazyLoad: false,
```

Comment:
This is fine. We've had to do this on all places we've used this lib for the same reason 😄

Reply:
Ok I'll remove the comment then 👍

Reply:
I thought we made the carousel `lazyLoad` prop false by default though... weird